packages/ngtools/webpack/BUILD

```diff
@@ -0,0 +1,82 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+
+licenses(["notice"])  # MIT
+
+load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "webpack",
+    srcs = glob(
+        include = [
+          "src/**/*.ts",
+        ],
+        exclude = [
+            "src/**/*_spec.ts",
+            "src/**/*_spec_helpers.ts",
+        ],
+    ),
+    data = glob(
+        include = [
+          "src/type_checker_bootstrap.js",
```

Comment:
Hadn't thought about it much, but that makes sense. I can't remove it here thought, because on this PR we're still using the old test setup for large specs, and they end up using the TS sources for ngtools/webpack. Should remove it in https://github.com/angular/angular-cli/pull/16976 though.